CaliptraTrademarkAudit.md

```diff
@@ -0,0 +1,70 @@
+
+# Caliptra Trademark Auditing Process
+
+This document will describe the specific requirements to be reviewed by a 3rd party and attest to Caliptra TAC that the requirements have been met.
+
+List of Topics to review for Trademark Compliance
+
+ * Each of these things will have to be reviewed for an integration to get approval of Trademark Usage
+
+## Caliptra IP
+
+* Show how the RTL was integrated into the device environment
+* Show a Run of the tool to verify the RTL Release Hash (in development)
+* Show the ROM hash from a release ROM hash
+* Show the internal Caliptra FW signing process used to sign Caliptra FW
+* Show what options were used to synthesize the RTL
+* Show input wires to RTL
+  * Clock
+  * Reset
+  * Power Good, Ready for fuses, Ready for FW, etc.
+  * Show how the Generic input/output wires are addressed
+
+## Caliptra Wrapper
+
+* Show how the RTL key is provided and ensure that it is not available to any FW.
```

Comment:
RTL Key -> Obfuscation Key

Reply:
This is requiring integrators to prove a negative...